apps/rs/README.md

```diff
@@ -0,0 +1,39 @@
+Halide Renderscript "blur" and "copy" samples
+===
+
+These samples demonstrates two supported modes of operation for Renderscript codegen:
+ - vectorized operations over interleavead RGBA images;
+ - single-element operations over planar images.
+
+Prerequisites
+---
+
+You will need to have Android NDK in the path.
+This app only runs on arm-based Android devices.
```

Comment:
Yes, it should be, but I started with arm and there are several places in codegen where I specified "arm" as target platform for llvm bitcode generation. I haven't looked at how RS bitcode looks like if you target x86, it actually might be the same and on-device codegen will do correct x86 generation.
This investigation is on to-do list.